policy/lib/tekton/pipeline.rego

```diff
@@ -44,6 +48,17 @@ pipeline_label_selector(pipeline) := value if {
 } else := value if {
 	# Labels from a Tekton Pipeline definition
 	value := pipeline.metadata.labels[pipeline_label]
+} else := value if {
```

Comment:
nit: is this last part needed? the fbc pipeline contains the label `pipelines.openshift.io/runtime` so the above two should find it if it's a valid fbc pipeline.

Reply:
Perhaps, I wasn't sure myself. It helps for very custom pipelines, though it might be an overhead that we don't need right now?

Reply:
Maybe just leave it. fbc seems like a special case.